### ISSUE TITLE
About PPT params results

### ISSUE THREAD

**Issue by SpeedyGonzales949** [Pointcept/Pointcept#392]
Hi! 

I was looking at the number of params for PPT and I find them intriguing. I noticed that PPT adds more params to SpUNet compared to PTv3. Why is that the case ? Is adaptive pdnorm disabled for PPT + PTv3 ? Also for the PPT + SpUNet did you keep affine true ? I am talking about this table:

| Model                | Params |
|----------------------|--------|
| SparseUNet       | 39.2M  |
| + PPT Sup. (f.t.)    | 41.0M  |
| PTv3           | 46.2M  |
| + PPT Sup. (f.t.)    | 46.3M  |

I am training now PPT + SparseUNet on ScanNet only, however I get 43M Params. The config I am using is this one: configs/scannet/semseg-ppt-v1m1-0-sc-st-spunet.py  

Could you please clarify ? I also trained the base SpUNet, and for that I get 39M params, as reported in the paper.

**Comment by Gofinge** [comment]
Yeah, here (https://github.com/Pointcept/Pointcept/blob/main/pointcept/models/point_prompt_training/prompt_driven_normalization.py#L23) The `decoupled=True` causes additional params.

Let me share with you more latest findings which will be public soon, hope this can be helpful:
1. Actually, BN causes the domain gap, so we need to decouple the normalization and add adaptive mechenisum.
2. LN is better in domain adaptation by will cause a performance drop in our ablation of PTv3.
3. So, if we insist in using LN to replace BN, and further scaling up. We can have a more elegant way to solve the gap issue mentioned in PPT.

**Comment by SpeedyGonzales949** [comment]
Thanks for sharing these insights. I am actually researching domain adaptation.  So these are quite helpful. But if  BN causes the domain gap why dont you drop every norm layer and let the prompts do the normalization ?

**Comment by Gofinge** [comment]
Drop every norm layer will definitely make the backbone perform very badly. We just recently successfully replaced all bn with ln and still have a good performance (after scaling up. BN still better when training with a small scale of data. But it won’t be a problem as we will provide pretrained backbone to our community)

**Comment by SpeedyGonzales949** [comment]
Thanks a lot again for the insights. Will check Sonata paper.
